src/locales/en.json

```diff
@@ -23,8 +23,8 @@
     "emergency_cta": "Call 911",
     "emergency_description": " Based on your reported symptoms, you should seek care immediately.",
     "emergency_title": "Call Emergency Services",
-    "experiencing_any_symptoms": "experiencing any of these symptoms",
-    "experiencing_symptoms": "experiencing at least one of these symptoms",
+    "not_experiencing_any_symptoms": "I am not experiencing any of these symptoms",
+    "experiencing_symptoms": "I am experiencing some of these symptoms",
     "i_am": "I am",
     "i_am_not": "I am not",
```

Comment:
If it is ok to join the two labels these two are not used anymore so we could delete it if you run `yarn i18n:extract` this process is made automatically